README.md

```diff
@@ -10,7 +10,7 @@
 ![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/guardrailsio/guardrails-cli?sort=semver)
 [![GuardRails CLI CI](https://github.com/guardrailsio/guardrails-cli/actions/workflows/ci.yaml/badge.svg)](https://github.com/guardrailsio/guardrails-cli/actions/workflows/ci.yaml)
 [![Go Report Card](https://goreportcard.com/badge/github.com/guardrailsio/guardrails-cli)](https://goreportcard.com/report/github.com/guardrailsio/guardrails-cli) 
-![Codecov](https://img.shields.io/codecov/c/github/guardrailsio/guardrails-cli?token=3c5e84bf-caa3-4a07-ace2-64f67b86a244)
+[![codecov](https://codecov.io/gh/guardrailsio/guardrails-cli/branch/master/graph/badge.svg)](https://codecov.io/gh/guardrailsio/guardrails-cli)
```

Comment:
Hi @gr-ben , shouldn't the `master` branch renamed to `main` ?

Reply:
yes, good catch!